src/pudl/metadata/fields.py

```diff
@@ -463,6 +463,11 @@
         "description": "Electric Plant In Service (USD).",
         "unit": "USD",
     },
+    "electric_plant_classification": {
+        "type": "string",
+        "description": "Status of electric plant (in service, future, leased, total).",
+        # "constraints": {"enum": ["in_service", "future", "total", "leased"]}
```

Comment:
Left these commented out on purpose, just as an indicator of what is in there. Maybe should remove or make more comments.

Reply:
yeah i would comment abt why if you want these here at all. I would personally not include them bc its pretty known that we should do some automatic enum-ing for all of the columns that get the `categorize_strings`

Also we've been using `{word_word}_type` for many of these category columns. I certainly won't die on the anti-classification hill, but I like the consistency of using a `type` suffix.

Reply:
Yeah maybe I should just delete

Reply:
or just `classification_type`

Reply:
Okay I went with `plant_classification_type` here just to be not utterly generic.